src/js/components/Button/StyledButton.js

```diff
@@ -34,9 +41,24 @@ const primaryStyle = props => css`
     props.theme,
     props.theme.button.color,
   )}
-  border-radius: ${props.theme.button.border.radius};
+  border-radius: ${radiusStyle(props)};
 `;
 
+const sizeStyle = props => {
```

Comment:
Can we rename this to `text-style`, since that's what it's controlling?

Reply:
Currently, it is also controlling `border.radius` and I was hoping to hear whether we want `pad` in as well, if we will include border and pad it wouldn't make sense to make it specific for `textStyle`.

Reply:
Rethinking this, I think I understand that your intention was just for the function name and not about the overall prop name. I agree and I'll address it.

Reply:
Done.